includes/widgets/heading.php

```diff
@@ -386,8 +386,7 @@ protected function render() {
 
 		$title_html = sprintf( '<%1$s %2$s>%3$s</%1$s>', Utils::validate_html_tag( $settings['header_size'] ), $this->get_render_attribute_string( 'title' ), $title );
 
-		// PHPCS - the variable $title_html holds safe data.
```

Comment:
Do we have this issue inside the heading widget only, or does it potentially apply to all widgets?

Reply:
It potentially could be applied to some of other widgets where sanitzing before render is not implemented.

Reply:
I understand. 

I assume there is nothing that we could do about that now, unless we would want to check the rendering of each widget.

I am leaving this comment here, so that Louis can give his input as well.

Reply:
Talking in Slack